### ISSUE TITLE
Do not use *interface{}

### ISSUE THREAD

**Issue by xrn** [awslabs/goformation#470]
SQS example:

```
RedrivePolicy *interface{} `json:"RedrivePolicy,omitempty"`
```

I think should be 

```
RedrivePolicy interface{} `json:"RedrivePolicy,omitempty"`
```

Probably this line is making a problem 
```
{{$prop_name}} {{if (or ($property.IsPolymorphic) ($property.IsCustomType) (not ($property.Required)) )}}*{{end}}{{$property.GoType $.Name $.Basename $name}} `json:"{{$name}}{{if (not ($property.Required))}},omitempty{{end}}"`
```

And should be improved

**Comment by rubenfonseca** [comment]
Hi @xrn, thanks for the ping. We are currently on summer holidays and unfortunately we don't have the bandwidth in the team to fix this. You can wait a couple of weeks, or submit a PR to fix it if you have the time. We appreciate your understanding!

**Comment by rubenfonseca** [comment]
Hi all, we've pushed a fix to 6.8.0 that should remove all pointers to interfaces! Thanks for your comments and feedback.
